db/virtual_tables.cc

```diff
@@ -0,0 +1,1030 @@
+/*
+ * Modified by ScyllaDB
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: (AGPL-3.0-or-later and Apache-2.0)
```

Comment:
I think this "Apache-2.0" and "Modified by ScyllaDB" are likely to be a mistake - I don't think anything in this file is from Cassandra, I think the virtual tables was our own invention. But please correct me if I'm wrong.

Reply:
Yes, this is copy-pasta. Will remove.